dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -5662,6 +5664,7 @@ private Contentlet internalCheckin(Contentlet contentlet,
             }
 
             contentlet = applyNullProperties(contentlet);
+            contentlet = isNewContent? setDefaultValues(contentlet):contentlet;
```

Comment:
Add a config property here to disable if needed.  This is introducing new behavior and we can't be sure that some other customer using the api is expecting it to work differently.

Reply:
Config lazy prop added, resolved